### ISSUE TITLE
 Loss is NaN issue

### ISSUE THREAD

**Issue by TaniMasanori** [shenghanlin/SeismicFoundationModel#6]
Hi, 
When I try to do Finetune using Network after Pretrain, Loss is NaN before Epoch 0 starts. Please let me know if there is a solution for this kind of problem.

Thank you for sharing your great work!

**Comment by shenghanlin** [comment]
In this case, you may need to check whether there are problems with your data, such as non-normalization, and problems with NaN values.

**Comment by TaniMasanori** [comment]
Thank you! Now, I can run the training.
